batchflow/models/torch/layers/combine.py

```diff
@@ -115,8 +115,15 @@ def spatial_resize(self, inputs):
             shape = get_shape(item)
             dim = get_num_dims(item)
             spatial_shape = shape[-dim:]
-            if dim > 0 and spatial_shape != tuple([1]*dim) and spatial_shape != spatial_shape_:
-                item = center_crop(item, get_shape(inputs[0])[2:])
+            if dim > 0 and spatial_shape != tuple([1] * dim) and spatial_shape != spatial_shape_:
+                # Handle dim >= 2
```

Comment:
What about fix for 3D? The function will not make shapes equal, only last two axes

Reply:
updated